packages/connect/src/core/index.ts

```
@@ -1160,6 +1161,12 @@ export class Core extends EventEmitter {
             case UI.LOGIN_CHALLENGE_RESPONSE:
                 this.uiPromises.resolve(message);
                 break;
+            case UI.RECEIVE_FIRMWARE:
+                const localFirmwares = message.payload && parseLocalFirmwares(message.payload);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)